tests/wicked/basic.pm

```diff
@@ -58,8 +58,8 @@ sub run {
     record_info('Test 7', 'Bring an interface up with wicked');
     assert_script_run("ifup $iface");
     assert_script_run("ping -q -c1 -W1 -I $iface 10.0.2.2");
-    # TODO - solve ticket https://progress.opensuse.org/issues/32248 and uncomment line
-    #validate_script_output("ip address show dev $iface", sub { m/(?=inet)(?=[dhcp])/g; });
+    validate_script_output("ip address show dev $iface", sub { m/inet/g; });
+    validate_script_output("wicked show dev $iface",     sub { m/[dhcp]/g; });
```

Comment:
@asmorodskyi - using `[dhcp]` is checking for presence of ANY of characters `d`, `h`, `c`, `p`
Check it on https://regex101.com/ - or am wrong somehow?

Reply:
@kravciak good catch , thanks ! 
I will create another PR which will fix this